skimage/measure/_structural_similarity.py

```diff
@@ -26,7 +26,7 @@ def compare_ssim(X, Y, win_size=None, gradient=False,
         window size will depend on `sigma`.
     gradient : bool, optional
         If True, also return the gradient with respect to Y.
-    data_range : int, optional
+    data_range : float, optional
```

Comment:
`int or float` maybe?

Reply:
is that really needed? I feel like if a field accepts a float, and int is _usually_ okay